Sources/StreamChatUI/Deprecations.swift

```diff
@@ -165,3 +166,55 @@ extension Components {
         }
     }
 }
+
+// MARK: - Deprecation of ChatMessageLayoutOptions as an OptionSet
+
+/// Previously `ChatMessageLayoutOptions` was an `OptionSet`, this limited the customization on
+/// the customer side because the raw value needs to be an `Int`. A more flexible approach is to just
+/// have a `Set` of `ChatMessageLayoutOption`. So for backwards compatibility we created the following
+/// typealias `typealias = Set<ChatMessageLayoutOption>` and provided an API like the `OptionSet` so we
+/// don't break the public API.
+
+extension ChatMessageLayoutOptions {
+    @available(*, deprecated, message: "use `id` instead.")
+    var rawValue: String {
+        id
+    }
+
+    @available(*, deprecated, message: "use `remove(_ member: ChatMessageLayoutOption)` instead.")
```

Comment:
@nuno-vieira Could we provide fix-its instead of deprecation messages?

Would something like the following work?
```suggestion
    @available(*, deprecated, renamed: "remove(_ member: ChatMessageLayoutOption)")
```

Reply:
Actually renamed is not needed in this case, since you provided the `typealias` that will route the calls towards the correct implementation of each respective func ;)

<img width="675" alt="Screenshot 2021-12-03 at 11 39 35" src="https://user-images.githubusercontent.com/4924709/144588933-038cf3da-dd55-4538-a1ac-09bd44425715.png">